testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -371,9 +371,15 @@ QUnit.test("Hide helper text when validation message shows for material theme",
         ]
     }).dxForm("instance");
 
-    form.validate();
     form.getEditor("lastName").focus();
+    form.validate();
+
+    triggerKeyUp(form.getEditor("lastName").$element(), "Enter");
```

Comment:
Can we use some variables for form.getEditor("lastName") or form.getEditor("lastName").$element()

Reply:
Done